README.md

```diff
@@ -7,6 +7,19 @@ This is the official LaunchDarkly client-side JavaScript SDK. This SDK does two
 * Makes feature flags available to your client-side (front-end) JavaScript code.
 * Sends click, pageview, and custom events from your front-end for A/B tests and analytics.
 
+## Browser Support
+
+The LaunchDarkly client-side JavaScript SDK supports the following browsers:
+
+* Chrome (any recent)
+* Firefox (any recent)
+* Safari (any recent)*
+* Internet Explorer (IE10+)*
```

Comment:
I only tested this in IE10. IE8,9 may be supported but I wasn't able to confirm.